spec/requests/headers_spec.rb

```diff
@@ -30,4 +30,27 @@
 
     expect(response.status).to eq 200
   end
+
+  context 'secure headers' do
+    it 'includes Strict-Transport-Security (HSTS)' do
+      get root_path
+
+      pending 'seems to not get set in plain http tests'
```

Comment:
Can we force it to request as `https:` ? **Edit:** Or is this also related to your original comment about `secure_headers` gem clobbering?

Reply:
The clobbering was mostly related to the `action_dispatch.default_headers` this was me being lazy... but with some Googling I found that this works: 9e1077b68411646c3ab61e77ce13a98c8ee656d5